### ISSUE TITLE
Last activity message for file uploading is too general

### ISSUE THREAD

**Issue by ta2-1** [translate/pootle#4297]
This is an example of last activity message when an user has uploaded a file:
`<user> uploaded file An hour ago`.
There is no information about name of the file. I think it can be confusing when we look at last activity message for the whole TP.
When we update store from FS via `update_stores` we track every submission and last activity message reflects a real last action related to a unit. I'd suggest to do the same for upload.

**Comment by iafan** [comment]
I agree. The fact that the file was upload doesn't help understand what was changed. If file uploading results in some translations being added or changed, then this is what we want to see in the last activity.

One improvement might be to show a `via file upload` suffix in the message, e.g.:

`John Doe edited Copy %@ 11 weeks ago via file upload`

**Comment by unho** [comment]
> One improvement might be to show a `via file upload` suffix in the message, e.g.:
>
> `John Doe edited Copy %@ 11 weeks ago via file upload`

+1

**Comment by phlax** [comment]
Following chat around this with @ta2-1 and @dwaynebailey it was proposed to:

- separate SubmissionType into SubmissionSource and SubmissionAction
- write a migration mapping existing types onto new types
- update ui to use Source and Action rather than Type
